package.json

```diff
@@ -101,6 +89,16 @@
     "react-select": "1.2.1",
     "react-virtualized": "9.18.5",
     "recompose": "^0.30.0",
-    "redux": "^4.0.1"
+    "redux": "^4.0.1",
+    "rxjs": "^6.4.0",
+    "sass": "^1.17.2",
+    "storybook": "^1.0.0",
+    "superagent": "^4.0.0",
+    "node-sass": "^4.10.0"
+  },
+  "dependencies": {
```

Comment:
The Server uses the package "serve-static". This should also be in dependencies?!

Reply:
Did you test that? `serve-static` should be a `node` core module.  
`npm i && npm prune --production && npm run build && npm run start` seems to work fine.

Reply:
Yes it is a dependency of express. But we now rely on a dependency from a dependency. What if express will drop serve-static in favour of another module? Our code will break.

If we don't want server-static as a direct dependency then in the grud-frontend-server.js file we should not require it directly but use `express.static` instead.

Reply:
I checked the code - you're right; I thought we had used express' core module, but we use a transient dependency of express. Added that.